### DISCUSSION TITLE
Method Parameters in java

### DISCUSSION THREAD

**Discussion by CadenCCC** [community/community#131634] [Programming Help]
### Body

How can I make my method have a parameter type that isnt `[]` but `...` ive been seeing them everywhere.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by 7orivorian** [comment]
Like so:
```java
public void myMethod(String... strings) {}
```

**Comment by ebndev** [comment]
Hi @CadenCCC & @7orivorian, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
